src/data/jetp-id/jetp-cipp-2023_timeseries.json

```diff
@@ -0,0 +1,1276 @@
+      "value": 50.1,
+      "unit": "GW"
+    },
+    {
+      "year": 2050,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "hydro",
+      "metric": "capacity",
+      "value": 65.4,
+      "unit": "GW"
+    },
+    {
+      "year": 2022,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "solar",
+      "metric": "capacity",
+      "value": 0.1,
+      "unit": "GW"
+    },
+    {
+      "year": 2025,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "solar",
+      "metric": "capacity",
+      "value": 4.1,
+      "unit": "GW"
+    },
+    {
+      "year": 2030,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "solar",
+      "metric": "capacity",
+      "value": 29.3,
+      "unit": "GW"
+    },
+    {
+      "year": 2035,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "solar",
+      "metric": "capacity",
+      "value": 77.1,
+      "unit": "GW"
+    },
+    {
+      "year": 2040,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "solar",
+      "metric": "capacity",
+      "value": 100.1,
+      "unit": "GW"
+    },
+    {
+      "year": 2045,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "solar",
+      "metric": "capacity",
+      "value": 177.6,
+      "unit": "GW"
+    },
+    {
+      "year": 2050,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "solar",
+      "metric": "capacity",
+      "value": 264.6,
+      "unit": "GW"
+    },
+    {
+      "year": 2022,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "wind",
+      "metric": "capacity",
+      "value": 0.1,
+      "unit": "GW"
+    },
+    {
+      "year": 2025,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "wind",
+      "metric": "capacity",
+      "value": 0.7,
+      "unit": "GW"
+    },
+    {
+      "year": 2030,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "wind",
+      "metric": "capacity",
+      "value": 8.6,
+      "unit": "GW"
+    },
+    {
+      "year": 2035,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "wind",
+      "metric": "capacity",
+      "value": 24.7,
+      "unit": "GW"
+    },
+    {
+      "year": 2040,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "wind",
+      "metric": "capacity",
+      "value": 29.2,
+      "unit": "GW"
+    },
+    {
+      "year": 2045,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "wind",
+      "metric": "capacity",
+      "value": 36.3,
+      "unit": "GW"
+    },
+    {
+      "year": 2050,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "wind",
+      "metric": "capacity",
+      "value": 44,
+      "unit": "GW"
+    },
+    {
+      "year": 2022,
+      "geography": "ID",
+      "sector": "power",
+      "technology": "Battery Storage",
```

Comment:
@jacobvjk The "Battery Storage" technology isn't in the enums list that we have in the schema. Should this file be changed, or the list in the schema updated?